tests/test_frontdoor.py

```diff
@@ -269,7 +272,7 @@ def _do_happy_flow(supplier="EON", benefits_answer="Yes", park_home=False, house
         supplier_shown = "E.ON Next"
     if supplier == "Shell":
         supplier_shown = "Octopus Energy"
-
+    print(page.path)
```

Comment:
```suggestion
```

Reply:
This change should already be in! I ran check-code locally and it didn't flag any changes to test_frontdoor for some reason. I've made the changes it asked for now